ambry-router/src/test/java/com.github.ambry.router/InMemoryRouter.java

```diff
@@ -42,16 +51,16 @@
   public final static String OPERATION_THROW_EARLY_RUNTIME_EXCEPTION = "routerThrowEarlyRuntimeException";
   public final static String OPERATION_THROW_LATE_RUNTIME_EXCEPTION = "routerThrowLateRuntimeException";
   public final static String OPERATION_THROW_ROUTER_EXCEPTION = "routerThrowRouterException";
-
-  private static final long BLOB_ID_SIZE = UUID.randomUUID().toString().length();
-
-  private final ConcurrentHashMap<String, InMemoryBlob> blobs = new ConcurrentHashMap<String, InMemoryBlob>();
-  private final ConcurrentSkipListSet<String> deletedBlobs = new ConcurrentSkipListSet<String>();
-
+  private static final long BLOB_ID_SIZE =
```

Comment:
Now have truly mimic the behavior of the nb router by deserializing a blobid string and throw `RouterException` if fails.